src/ComposerManager.php

```diff
@@ -56,11 +67,14 @@ public function getPackageForLibrary(string $library): PackageInterface
 
     /**
      * Return all the packages installed in the current setup.
-     * @return array Array of \Composer\PackageInterface
+     * @return array<Composer\PackageInterface>
      */
     public function getPackages(): array
     {
-        return $this->getComposer()->getRepositoryManager()->getLocalRepository()->getPackages();
+        return array_merge(
+            [$this->getComposer()->getPackage()],
```

Comment:
Yes, the root package is not included, only it's dependencies. Composer->getPackage() returns the root package, thus fixing this.